app/src/types/genericTypes.js

```diff
@@ -5,24 +5,51 @@
 import t from "flow-runtime";
 import * as v from "flow-runtime-validators";
 
+/**
+ * A string that represents a valid italian Fiscal Number.
+ */
 export const FiscalNumberType = t.refinement(
   t.string(),
   v.regexp({
     pattern: /^[A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]$/
   })
 );
 
+/**
+ * A string that represents a valid email address.
+ */
 export const EmailType = t.refinement(t.string(), v.email());
 
+/**
+ * A string that represents a number greater than or equal to zero.
```

Comment:
this is a refinement of `number`, it's not a `string`, it's a `number` -> `A number greater than or equal to zero.`

Reply:
ops, to much copy and paste, I'll fix them